Beyond06-SOW.md

```diff
@@ -0,0 +1,344 @@
+| 3              | 31.10                  | Databases, data model and the ORM                                                                                                                                                                                                                            |  Sim Zacks           |
+| 4              | 07.11                  | Demo 1: Development environment           |                      |
+
+| 5              | 14.11                  | Application logic and Testing                                                                                                                                                                                                                                |   Edy Haas           |
+| 6              | 21.11                  | UI: Routes, Views and Templates                                                                                                                                                                                                                              |   Yariv              |
+| 7              | 28.11                  | Demo 2: Data Model                                                                                                                                                                                                                                           |                      |
+|                | Nov 28 - Dec 02        | Performance review week                                                                                                                                                                                                                                      |                      |
+| 8              | 5.12                   | Going to production 1 - PAAS and containers                                                                                                                                                                                                                  | Barak Korren         |
+
+|                |                        | Writing and testing the Django application views and templates, bootstrapping (navigation), Practicing testing                                                                                                                                               |  Yariv Rachmany      |
+| 9              | 19.12                  | Demo 3: Testing, Control logic and UI                                                                                                                                                                                                                        |                      |
```

Comment:
Lets keep the demo names consistent between here, the list above and the spreadsheet.

Same goes for lecture names

Reply:
I meant for the whole list and table, not just this line - in any case, not seeing the change you've mentioned